scripts/end_to_end_test.py

```diff
@@ -36,6 +36,22 @@
 logger = logging.getLogger(__name__)
 
 
+def setup_environment_variables():
+    """Set up environment variables with staging fallbacks if needed"""
+    staging_api_key = os.getenv("STAGING_VELLUM_API_KEY")
+    if not os.getenv("VELLUM_API_KEY") and staging_api_key:
+        os.environ["VELLUM_API_KEY"] = staging_api_key
+        logger.info("Using STAGING_VELLUM_API_KEY as VELLUM_API_KEY")
```

Comment:
we don't need these env vars

Reply:
Sorry I meant we don't need these logger info statement. Bring the env vars back